setup.py

```diff
@@ -267,7 +267,7 @@ def validate_and_update_archs(archs):
     if not os.path.exists("./build"):
         os.makedirs("build")
 
-    os.system(f"python3 {ck_dir}/example/ck_tile/01_fmha/generate.py --output_dir build --receipt 2")
+    os.system(f"{sys.executable} {ck_dir}/example/ck_tile/01_fmha/generate.py --output_dir build --receipt 2")
```

Comment:
Could you fix it for both fwd and bwd?  setup.py already update now

Reply:
done